stylesheets/styles.css

```diff
@@ -0,0 +1,24 @@
+.rating {
+  unicode-bidi: bidi-override;
+  direction: rtl;
+  font-size: 30px;
+}
+.rating .star {
+  font-family: 'FontAwesome';
+  font-weight: normal;
+  font-style: normal;
+  display: inline-block;
+}
+.rating .star:hover {
+  cursor: pointer;
+}
+.rating .star:before {
+  content: "\f006";
+  padding-right: 5px;
+  color: #777;
+}
+.rating .star:hover:before,
+.rating .star:hover ~ .star:before {
+  content: "\f005";
+  color: #e3cf7a;
+}
```

Comment:
This entire file is a duplicate of what's in /assets/stylesheets/styles.css and should prob be nuked.